index.js

```diff
@@ -40,6 +40,10 @@ function isBuffer (x) {
   return true;
 }
 
+function isError (x) {
+  return x instanceof Error;
```

Comment:
The main issue is that this won’t work across realms, and it will fail on an Error that has had its [[Prototype]] changed.